src/OpenLoco/src/Ui/Widgets/PanelWidget.cpp

```diff
@@ -15,27 +15,31 @@ namespace OpenLoco::Ui::Widgets
             return;
         }
 
-        if (window->minHeight == window->maxHeight && window->minWidth == window->maxWidth)
+        if (window->minHeight == window->maxHeight || window->minWidth == window->maxWidth)
```

Comment:
This is reminded me of #2928, but it looks like the bug was reintroduced?? Nice catch.

Reply:
There were two places with this check so its not a regression.

Reply:
Odd, I seem to have modified the exact same line the other day.